packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -408,9 +408,10 @@ abstract class TestRunner {
   );
 
   private expectedLimboDocs: DocumentKey[];
-  private expectedActiveTargets: {
-    [targetId: number]: { queries: SpecQuery[]; resumeToken: string };
-  };
+  private expectedActiveTargets: Map<
```

Comment:
This type is repeated several times in here. Would it make sense to create a type alias for the map type or even just the value type within the map?